### ISSUE TITLE
No additional images are shown when a user post in a quick reply

### ISSUE THREAD

**Issue by Eldenroot** [Destroy666x/MyBB-Animated-Additional-Images#1]
No additional images are shown (in a new post) when a user post it in a quick reply until this user refresh the page.

**Comment by Destroy666x** [comment]
Does that happen with plugin enabled in previews?

If it does, try changing:

``` php
if(THIS_SCRIPT == 'showthread.php' && $mybb->settings['animated_additional_images_posts'])
```

to:

``` php
if(THIS_SCRIPT == 'showthread.php' || THIS_SCRIPT == 'newreply.php' && $mybb->settings['animated_additional_images_posts'])
```

and let me know if it works.

**Comment by Eldenroot** [comment]
I will try in 30 minutes

**Comment by Destroy666x** [comment]
Ok. I also pushed a quick pause implementation test: https://github.com/Destroy666x/MyBB-Animated-Additional-Images/commit/41527cfd9843ad4483f04eb55f0fbb5419bd82c5 if you want to test it too - reinstallation is required.

I go sleep now and I will test it myself and maybe release a new stable version tommorow.

**Comment by Eldenroot** [comment]
Yes, now it works fine with your change above.

Pause feature is also fine, so thank you very much! now I am satisfied ;)

**Comment by Destroy666x** [comment]
Fixed in 1.3.
